fs/log/log.go

```diff
@@ -117,17 +124,35 @@ func InitLogging() {
 
 	// Log file output
 	if Opt.File != "" {
-		f, err := os.OpenFile(Opt.File, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0640)
-		if err != nil {
-			log.Fatalf("Failed to open log file: %v", err)
-		}
-		_, err = f.Seek(0, io.SeekEnd)
-		if err != nil {
-			fs.Errorf(nil, "Failed to seek log file to end: %v", err)
+		var logWriter io.Writer
+		if Opt.Rotate {
+			logWriter = &lumberjack.Logger{
+				Filename:   Opt.File,
+				MaxSize:    int(Opt.RotateMaxSize) / 1024 / 1024, // megabytes
+				MaxAge:     Opt.RotateMaxAge,
+				MaxBackups: Opt.RotateMaxBackups,
+				LocalTime:  false,
+				Compress:   Opt.RotateCompress,
+			}
+			// Not ideal, but lumberjack doesn't support a way to get the underlying file
```

Comment:
This is the only problematic part since we don't have access to the underlying file and hence, no way to redirect stderr.

Reply:
I didn't like the approach but I couldn't find anything better so I left it commented as just an approach idea ... 😅 

If you feel it's necessary, I can uncomment it.